mypy/errors.py

```diff
@@ -553,12 +563,11 @@ def is_errors(self) -> bool:
 
     def is_real_errors(self) -> bool:
```

Comment:
This method seems unused, maybe we should just remove it?

Reply:
done